workers/ohsome_quality_analyst/indicators/mapping_saturation/indicator.py

```diff
@@ -124,12 +126,16 @@ def calculate(self) -> None:
             self.result.class_ = 3
         elif self.lower_threshold >= self.result.value > 0:
             self.result.class_ = 1
-        else:
```

Comment:
I would keep the else statement at the end of the elif statements to catch unexpected values in case they occure